core/parameters/res/runtime_configs/parameters.snap

```diff
@@ -208,7 +208,7 @@ vm_kind                                 NearVm
 eth_implicit_accounts                   true
 yield_resume                            true
 discard_custom_sections                 true
-max_congestion_incoming_gas             40_000_000_000_000_000
+max_congestion_incoming_gas             400_000_000_000_000_000
```

Comment:
Can we edit this file? There is a note in 3rd line that we should not edit it :thinking: